tests/test_sft_trainer.py

```diff
@@ -1377,6 +1377,66 @@ def test_run_moe_ft_and_inference(dataset_path):
         )
 
 
+@pytest.mark.skipif(
+    not is_fms_accelerate_available(plugins="moe"),
+    reason="Only runs if fms-accelerate is installed along with accelerated-moe plugin",
+)
+@pytest.mark.parametrize(
+    "dataset_path",
+    [
+        TWITTER_COMPLAINTS_DATA_JSONL,
+    ],
+)
+def test_run_moe_ft_and_inference_kernels_only(dataset_path):
```

Comment:
In general just advisable to add screenshots of the successful test cases runs in the PR description with this path enabled if you add gated test cases.

Reply:
<img width="846" alt="Screenshot 2025-04-02 at 10 02 24 PM" src="https://github.com/user-attachments/assets/f31550fd-e024-4b6b-a985-287042100ac5" />

There you go!